spec/support/formatter_support.rb

```diff
@@ -1,21 +1,8 @@
 module FormatterSupport
-  def run_example_specs_with_formatter(formatter_option, normalize_output=true)
-    options = RSpec::Core::ConfigurationOptions.new(%W[spec/rspec/core/resources/formatter_specs.rb --format #{formatter_option} --order defined])
+  def run_example_specs_with_formatter(formatter_option, options={}, &block)
+    output = run_rspec_with_formatter(formatter_option, options.merge(:extra_options => ["spec/rspec/core/resources/formatter_specs.rb"]), &block)
 
-    err, out = StringIO.new, StringIO.new
-    err.set_encoding("utf-8") if err.respond_to?(:set_encoding)
-
-    runner = RSpec::Core::Runner.new(options)
-    configuration = runner.configuration
-    configuration.backtrace_formatter.exclusion_patterns << /rspec_with_simplecov/
-    configuration.backtrace_formatter.inclusion_patterns = []
-
-    yield runner if block_given?
-
-    runner.run(err, out)
-
-    output = out.string
-    return output unless normalize_output
+    return output unless options.fetch(:normalize_output) { true }
```

Comment:
`fetch(:normalize_output, true)` could be used here, seems pointless instantiating a block here...